index.js

```diff
@@ -83,6 +84,10 @@ module.exports = terria.start({
             // new GnafSearchProviderViewModel({terria})
         ];
 
+       //  viewState.errorHandler = new RollbarErrorProvider({
```

Comment:
In the instance of config being loaded from magda, have you checked whether configParameters correctly reflects the `rollbarAccessToken` after the `terria.start` call?

Reply:
(I would think that yes it does but i'm not 100%)